Add warning on V9 CHANGELOG (#2586)
RevenueCat/purchases-android: CHANGELOG.md

## 9.0.0
## RevenueCat SDK

> [!WARNING]  

> > If you don't have any login system in your app, please make sure your one time products have been correctly configured in the RevenueCat dashboard as non-consumables if needed. If not, our SDK will consume those purchases, and they won't be able to be restored starting on this version.

Reviewer: ```suggestion
> If you don't have any login system in your app, please make sure your one-time purchase products have been correctly configured in the RevenueCat dashboard as non-consumables if needed. If not, our SDK will consume those purchases, and they won't be able to be restored starting on this version.
```

> > If you don't have any login system in your app, please make sure your one-time purchase products have been correctly configured in the RevenueCat dashboard as non-consumables if needed. If not, our SDK will consume those purchases, and they won't be able to be restored starting on this version.

Reviewer 2: Maybe the "if needed" is superfluous? "One-time purchase" already implies they should not be consumable? 

```suggestion
> If you don't have any login system in your app, please make sure your one-time purchase products have been correctly configured in the RevenueCat dashboard as non-consumables. If not, our SDK will consume those purchases, and they won't be able to be restored starting on this version.
```

Author: Actually, one-time purchase means either consumable or non-consumable... Agree it can be confusing, but that's the naming Google gave it with BC8 🥴

Reviewer 2: Ouch, that doesn't help haha. Okay how about: 

>[...] please make sure your one-time purchase products have been correctly configured in the RevenueCat dashboard as either consumable or non-consumable.

(GitHub doesn't allow me to create a suggestion because this thread is outdated.)

> > If you don't have any login system in your app, please make sure your one-time purchase products have been correctly configured in the RevenueCat dashboard as non-consumables if needed. If not, our SDK will consume those purchases, and they won't be able to be restored starting on this version.

Reviewer 2: ```suggestion
> If you don't have any login system in your app, please make sure your one-time purchase products have been correctly configured in the RevenueCat dashboard as non-consumables if needed. If they're incorrectly configured as consumables, RevenueCat will consume these purchases. This means that users won't be able to restore them from version 9.0.0 onward.
```
